lib/deprecate_delegates.ml

```diff
@@ -0,0 +1,37 @@
+let warning =
+  "dune-release delegates are deprecated. They will be removed in version \
+   2.0.0."
```

Comment:
I think we should add pointers to the replacement solution in the warnings. Somehting like:

"You should write scripts invoking dune-release instead. Note that if you require some values that used to be passed by dune-release to your delegate you can obtain them with the `dune-release delegate-info` command added in 1.4.0"

I'm not sure if it should appear everywhere but at least for all the big entry points so the doc and the logs I guess.

Reply:
Yes, that's a good idea. I've done that.